File: data-quality/trash/RoleCheckerTest.scala
package it.gov.daf.dataquality.analysis.email.checkers

import org.scalatest.FunSuite

/**
  * @author Marco Bianchi (bianchi74@gmail.com)
  */
class RoleCheckerTest extends FunSuite {

  test("testGetInstance") {
    RoleChecker.getInstance()
  }

  test("testHasRole"){
    val checker = RoleChecker.getInstance()
    assert(checker.isChecked("info@"))
    assert(!checker.isChecked("notARole@"))
  }

}

File: data-quality/src/test/java/it/gov/daf/dataquality/analysis/email/checkers/RoleCheckerTest.java
package it.gov.daf.dataquality.analysis.email.checkers;

import junit.framework.TestCase;
import org.junit.Test;

/**
 * @author Marco Bianchi (bianchi74@gmail.com)
 */
public class RoleCheckerTest extends TestCase {

    @Test
    public void testIsChecked(){
        Checker checker = RoleChecker.getInstance();
        assert(checker.isChecked("info@"));
        assert(!checker.isChecked("notARole@"));
    }
}
